src/stylesheets/Book.css

```diff
@@ -0,0 +1,75 @@
+.Book {
+  display: flex;
```

Comment:
- [ ] [REQUIRED] In the stylesheets of the app, make sure that basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices) ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- no repeating of the same rule.